packages/gatsby/cache-dir/normalize-page-path.ts

```diff
@@ -1,4 +1,4 @@
-export default path => {
+export default (path: string): string => {
```

Comment:
let's avoid default exports and use named one (and adjust imports to it) - same apply to second copy of the file

Reply:
Should be fixed now. By the way, which test method would you recomment locally? `yarn test` takes quite some time, and `yarn jest gatsby` seems to result in different errors.

Reply:
both should be fine - as long as you run them from root of monorepo

I'll take a look at currently failing tests

Reply:
Ah thanks! 

Looks like there is some issue with the import in `cache-dir` since there is no typescript code in there. JS probably doesn't like importing TS in that folder. I couldn't find a reason, also, looks like it's the only file that was copied for duplication in that folder. I'm not that experienced yet with these kinds of imports :)

Reply:
@pieh did you get any chance to look at the tests?

Reply:
I just pulled it down locally, so not yet, but hope in nearest 1 hour to have some news

Reply:
this should be fixed (we weren't building .ts files in `cache-dir` directory)